hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java

```diff
@@ -254,6 +255,16 @@ public TransactionInfo runAllChecks(
         return txInfo;
     }
 
+    private static int getTransactionMaxSize(Configuration configuration) {
```

Comment:
This is a little misleading. A transaction's max size should also depend on the type of the transaction, right? We don't want to allow jumbo transactions for any type

Reply:
Yes, the max size depends on the transaction type but in this case, this is a max size for the "fail fast" check inside `parseAndCheck`. The idea is to fail even before we parse the transaction. Later on, when we do parse it we check the transaction type as well.

I am open to a better method name if you have suggestions.

Reply:
Yeah, kind of tough to name 🤔 Maybe 

```suggestion
    private static int maxIngestParseSize(Configuration configuration) {
```

Maybe that's not really better. Not a huge deal though.

Reply:
I like it, changed 👍